packages/commonwealth/client/scripts/views/pages/MyCommunityStake/Stakes/Stakes.tsx

```diff
@@ -87,10 +92,7 @@ const Stakes = ({ transactions }: TransactionsProps) => {
       .map((transaction: any) => ({
         ...transaction,
         voteWeight: transaction.voteWeight + 1, // total vote weight is +1 of the stake weight
-        avgPrice: `${transaction.avgPrice.toFixed(5)} ${getCommunityStakeSymbol(
-          app.config.chains.getById(transaction?.community?.id)?.ChainNode
-            ?.name || '',
-        )}`,
+        avgPrice: `${transaction.avgPrice.toFixed(5)} ${'ETH'}`,
```

Comment:
i think this should be dynamic per, similar to what we are doing in https://github.com/hicommonwealth/commonwealth/pull/7205

Reply:
@mzparacha I agree, let me change it. I didn't like this solution so thank you for the suggestion.

Reply:
@mzparacha After speaking with Zak Hap, as of now at no point will either BASE or Blast not be ETH, therefore keeping `ETH` hardcoded for now is fine. When we expand and allow other ways to buy stake that will require a bigger build out and refactoring of this line of code.

Reply:
i see, i think we can resolve this comment then. Thanks